### PR TITLE
[Merge on April 28th] Add release notes for Magento Cloud Patches 1.0.3

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Updated the Magento Cloud Patches release notes with information about the latest release (1.0.3). 

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/release-notes/cloud-tools.html
- https://devdocs.magento.com/cloud/release-notes/mcp-release-notes.html

## Links to Magento source code

https://github.com/magento/magento-cloud-patches/pull/30

whatsnew
Published the release notes for [Magento Cloud Patches 1.0.3](https://devdocs.magento.com/cloud/release-notes/mcp-release-notes.html).

### CODE REVIEW

**Comment by meker12** [comment]
Publish the changes in this PR in conjunction with the v2.3.5 release.

File: src/cloud/release-notes/mcp-release-notes.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ The `{{site.data.var.mcp}}` package uses the following version sequence: `<major
 {:.bs-callout-info}
 See [Apply patches]({{site.baseurl}}/cloud/project/project-patch.html) to learn how to apply Magento patches and hot fixes to your {{site.data.var.ece}} project.
 
+## v1.0.3
+*Release date: April 28, 2020*<br/>
+
+-  Added the "FPC is getting disabled during deployments" patch to support Magento 2.3.5.
```
Comment (YPyltiai) [inline src/cloud/release-notes/mcp-release-notes.md:22]: Please change "Added the" for "Added fix for the"

Comment (meker12) [inline src/cloud/release-notes/mcp-release-notes.md:22]:
```suggestion
-  Added fix for the "FPC is getting disabled during deployments" patch to support Magento 2.3.5.
```

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**